cmd/openfga/openfga.go

```diff
@@ -56,6 +59,10 @@ type svcConfig struct {
 	// RequestTimeout is a limit on the time a request may take. If the value is 0, then there is no timeout.
 	RequestTimeout time.Duration `default:"0s" split_words:"true"`
 
+	EnableTLS   bool   `default:"false" split_words:"true"`
+	TLSCertPath string `split_words:"true"`
+	TLSKeyPath  string `split_words:"true"`
```

Comment:
Please submit another PR here: https://github.com/openfga/openfga.dev/blob/main/docs/content/intro/setup-openfga.mdx#configuring-the-server

Reply:
Thanks! Will do this now.